Project.toml

```
@@ -24,7 +24,7 @@ AWS = "1.63.1"
 ArrowTypes = "1.2"
 Compat = "3.29.0"
 EzXML = "0.9, 1"
-FilePathsBase = "=0.9.11, =0.9.12, =0.9.13, =0.9.14, =0.9.15"
+FilePathsBase = "0.9"
```

```suggestion
FilePathsBase = "0.9.11"
```
Might as well set a minimum rather than re-allowing older versions, especially if we're all the way up at 0.9.18. This is what it was at before the change to this version of compat.